include/zephyr/posix/signal.h

```diff
@@ -67,6 +67,16 @@ typedef struct {
 #define SIGEV_THREAD 3
```

Comment:
Typo in commit message of `posix: pthread: implement pthread_sigmask()`
s/sitting/setting/